include/seastar/core/file.hh

```diff
@@ -100,6 +100,13 @@ struct file_open_options {
     static constexpr uint32_t min_extent_size_hint_alignment{128u << 10}; // 128KB
 };
```

Comment:
> Add APIs to get group details and to change ownership of file. This w…

Please have just one sentence in the subject, use `git log` to see the style of our commitlogs.